### PR TITLE
long names no longer slip down from the like-symbol, fixes #391

### LINKED ISSUE #391: UI: if list of likers doesn't fit on one line, the list gets displayed one line below like icon
Expected: 
image

Actual:
image

### PR DESCRIPTION
**Pull request by dym-cx**
test on post: https://micropeppa.freefeed.net/berkus/24f03122-9edd-4ff8-a400-50dfb63f5687

### CODE REVIEW

File: themes/helvetica/post.scss [hunk 1]
```diff
@@ -59,6 +59,7 @@
       padding: 0;
       margin: 0 4px 4px 21px;
       float: left;
+      max-width: 590px;
```
Comment (innin) [inline themes/helvetica/post.scss]: image
Hmm, it's not working for me in latest Chrome under mac os x

**Comment by dym-cx** [comment]
ok, 555px is better looking

**Comment by indeyets** [comment]
got confirmation from @innin in slack. he tells it works fine

**Comment by epicmonkey** [comment]
This PR doesn't fix issue completely, i.e. 555px is width when a right column is displayed (that happens for authorized users), however if I'm a not authorized user then max-width will be bigger.

**Comment by dym-cx** [comment]
ok, `width: 90%` fixes issue on all resolutions: http://imgur.com/a/v8IGE, PR #400

**Pull request merged**